ocp_install_env.sh

```diff
@@ -10,7 +10,7 @@ export SSH_PUB_KEY="`cat $HOME/.ssh/id_rsa.pub`"
 export SSH_PRIV_KEY="$HOME/.ssh/id_rsa"
 
 # Temporary workaround pending merge of https://github.com/openshift/machine-api-operator/pull/246
-export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="quay.io/vrutkovs/kni-release-mar13:release"
+export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="registry.svc.ci.openshift.org/ci-op-r5vsk4k4/release:latest"
```

Comment:
```suggestion
export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="quay.io/vrutkovs/kni-release:mar14"
```
The same release mirrored to quay, most likely its correct and these images won't be GC'ed

Reply:
@hardys is going to submit a separate PR with that